python/aswfdocker/builder.py

```diff
@@ -0,0 +1,110 @@
+import logging
```

Comment:
This should have an SPDX string to indicate its license, maybe a brief comment to indicate what it does?